tests/spec/rainforest/show_edit_remove_credentials.rfml

```diff
@@ -15,15 +15,15 @@ Click on 'amazon web services' and then click on 'CREATE STACK' button
 Has that pop-up disappeared? Do you see '_Aws Stack' title? Do you see 'Custom Variables', 'Readme' tabs? Do you see 'Credentials' text having a red (!) next to it? Do you see '# Here is your stack preview' in the first line of main content? Do you see 'DELETE THIS STACK TEMPLATE', 'PREVIEW' and 'LOGS' buttons on the bottom and 'SAVE' button on top right?
 
 Click on 'Credentials' and then click on 'ADD A NEW CREDENTIAL' button on the bottom
-Do you see 'Add your AWS credentials' title? Do you see 'Title', 'Access Key ID' , 'Secret Access Key' and 'Region' fields? Do you see 'CANCEL', 'ADVANCED MODE' and 'SAVE' buttons?
+Do you see 'Add your AWS credentials' title? Do you see 'Title', 'Access Key ID' , 'Secret Access Key' and 'Region' fields? Do you see 'CANCEL', 'ADVANCED MODE' and 'SAVE' buttons? (If buttons are not visible, scroll down until you see them)
 
 Enter "{{ random.first_name }}" in 'Title', "{{ random.number }}" in 'Access Key ID', "{{ random.password }}" in 'Secret Access Key' fields and click on 'SAVE' button below
-Is it added to the list above 'rainforestqa99's AWS keys'?
+Is it added to the top of the list above 'rainforestqa99's AWS keys'?
```

Comment:
What if `{{random.first_name}}` is substituted with something that might be listed under `rainforestqa99's AWS keys`?